superset/connectors/druid/models.py

```diff
@@ -126,14 +126,19 @@ def get_base_coordinator_url(self):
             self.coordinator_host, self.coordinator_port)
         return '{base_url}/{self.coordinator_endpoint}'.format(**locals())
 
+    def get_base_broker_url(self):
+        base_url = self.get_base_url(
+            self.broker_host, self.broker_port)
+        return '{base_url}/{self.broker_endpoint}'.format(**locals())
+
     def get_pydruid_client(self):
         cli = PyDruid(
             self.get_base_url(self.broker_host, self.broker_port),
             self.broker_endpoint)
         return cli
 
     def get_datasources(self):
-        endpoint = self.get_base_coordinator_url() + '/datasources'
+        endpoint = self.get_base_broker_url() + '/datasources'
```

Comment:
I think `get_base_coordinator_url` can now be removed. Looks like it's also referenced in `tests/druid_tests.py`.